HappyCamps/src/app/login/login.component.ts

```diff
@@ -0,0 +1,51 @@
+import { HttpClient } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+import { UntypedFormBuilder, UntypedFormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { AuthService } from '../services/auth.service';
+import { UserStoreService } from '../services/user-store.service';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent {
+  validateForm!: UntypedFormGroup;
+
+  constructor(private fb: UntypedFormBuilder,private auth:AuthService,private router:Router,private userStore:UserStoreService) {}
+
+  submitForm(): void {
+    if (this.validateForm.valid) {
+      this.auth.login(this.validateForm.value).subscribe({
+        next:(res)=>{
+          this.auth.storeToken(res.token)
```

Comment:
this seem like it should be inside a Service

Reply:
Why?